qpmodel/optimizer.cs

```diff
@@ -597,7 +590,7 @@ public CGroupMember CalculateMinInclusiveCostMember(PhysicProperty required = nu
 
             // check if we already done the calculation
             if (required == null && minMember_.ContainsKey(new PhysicProperty()))
```

Comment:
if we make required default value to PhysicProperty.nullprop (so never nul) , will this simplify this logics?

Reply:
the default value for optional parameter cannot be PhysicProperty.nullprop since it has to be compile time constant, I used overload as a bypass